src/ui/metatilelayersitem.cpp

```diff
@@ -19,11 +19,9 @@ void MetatileLayersItem::draw() {
         QPoint(80, 16),
     };
 
-    bool isTripleLayerMetatile = projectConfig.getTripleLayerMetatilesEnabled();
-    int width = isTripleLayerMetatile ? 96 : 64;
-    QPixmap pixmap(width, 32);
+    int numTiles = projectConfig.getNumTilesInMetatile();
+    QPixmap pixmap(numTiles * 8, 32);
```

Comment:
`numTiles * 8` doesn't make sense here, even though it's the correct value.  The pixmap is scaled to 2x here, so this should probably be rewritten as `(numTiles / 2) * 16`, or create a temporary variable called `numTilesWide = numTiles / 2`.

Reply:
Or maybe `projectConfig.getNumLayersInMetatile() * 32`

Reply:
Ah, I had 8 pixels wide per tile in my head